### PR TITLE
PR for Issue 1422: Externalize ID attributes for JWT builder and consumer

### LINKED ISSUE #1422: The "id" metatype in both jwtBuilder and jwtConsumer in jwt-1.0 feature need to be public
The configuration attribute is currently marked as "internal", while knowledge center document it which implies it is public.
Here is excerpt from knowledge center
"
If you specify a configuration ID, the object is tied to the jwtBuilder configuration with the specified ID.
com.ibm.websphere.security.jwt.JwtBuilder jwtBuilder = JwtBuilder.create("jwtBuilder_configuration_id");
".

### PR DESCRIPTION
**Pull request by ayoho**
Fixes #1422

### CODE REVIEW

File: dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -17,8 +17,10 @@
 
 jwtBuilder=JWT Builder
 jwtBuilder.desc=Information about configuring the builder. The elements and attributes that you specify are used to build the token.
-id=Builder id
-id.desc=This id is used to identify the builder.
+
+jwtBuilderId=Builder id
+jwtBuilderId.desc=This id is used to identify the JWT builder. If an id value is not specified, the builder will not be processed. The id must be a URL-safe string. The id is used as part of the issuer value if the issuer configuration attribute is not specified. The JwtBuilder API uses this id to determine which builder configuration to use to construct JWTs.
```
Comment (rhpeters) [inline dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties]: recommend changing "will not be processed" to "is not processed"

File: dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -68,6 +70,9 @@ sharedKey.desc=Specifies the string that will be used to generate the shared key
 jwtConsumer=JWT Consumer
 jwtConsumer.desc=The JWT consumer information to validate the JWT token.
 
+jwtConsumerId=Consumer id
+jwtConsumerId.desc=This id is used to identify the JWT consumer. If an id value is not specified, the consumer will not be processed. The id must be a URL-safe string. The JwtConsumer API uses this id to determine which consumer configuration to use to consume JWTs.
```
Comment (rhpeters) [inline dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties]: recommend changing "will not be processed" to "is not processed"

Comment (rhpeters) [reply dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties]: Recommend changing all instances of "id" to "ID"

File: dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (rhpeters) [reply dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties]: Recommend changing all instances of "id" to "ID"

**Comment by ayoho** [comment]
#build

**Comment by ayoho** [comment]
#build

**Comment by tevans78** [comment]
PB since been deleted but merging anyway in time for the next translation drop.

**Pull request merged**